org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/symbols/LSPSymbolInWorkspaceDialog.java

```diff
@@ -185,4 +189,26 @@ protected Control createExtendedContentArea(Composite parent) {
 		return null;
 	}
 
+	private static List<WorkspaceSymbol> toWorkspaceSymbols(List<? extends SymbolInformation> source) {
+		return source == null ?
+				List.of() :
+				source.stream().map(LSPSymbolInWorkspaceDialog::toWorkspaceSymbol).collect(Collectors.toList());
```

Comment:
what if `toWorkspaceSymbol` returns null? Should we not filter null objects before collecting to a list?